api/src/main/scala/hmda/api/http/LarHttpApi.scala

```diff
@@ -46,21 +48,23 @@ trait LarHttpApi extends LarProtocol with ValidationResultProtocol with HmdaCust
   val validateLarRoute =
     pathPrefix("lar") {
       path("validate") {
-        parameters('check.as[String] ? "all") { (checkType) =>
-          timedPost {
-            entity(as[LoanApplicationRegister]) { lar =>
-              val larValidation = system.actorSelection("/user/larValidation")
-              val checkMessage = checkType match {
-                case "syntactical" => CheckSyntactical(lar, ValidationContext(None))
-                case "validity" => CheckValidity(lar, ValidationContext(None))
-                case "quality" => CheckQuality(lar, ValidationContext(None))
-                case _ => CheckAll(lar, ValidationContext(None))
-              }
-              onComplete((larValidation ? checkMessage).mapTo[List[ValidationError]]) {
-                case Success(xs) =>
-                  complete(ToResponseMarshallable(xs))
-                case Failure(e) =>
-                  complete(HttpResponse(StatusCodes.InternalServerError))
+        hmdaAuthorize {
```

Comment:
Everything `ValidationContext`-related is not in scope of this PR. Please make a separate issue for that.